career/ladder.md

```diff
@@ -221,6 +255,12 @@ Here are some examples of other career ladders:
 - [ ] Anticipates technical issues at a product level and works with design to avoid them.
 
 ### Level 6 - Principal Software Engineer
+As a Principal Software Engineer, you play a critical role in how engineering at
+Abtion operates, and throughout your long career at Abtion, you have established
```

Comment:
At this level, we also allow an area of ownership. E.g. tooling, security, design, process.
We need people at this level to be drivers. They are not only technical, they push Abtion forward.
I cannot find the right wording. We can pair on this one, as it is the toughest one.

```suggestion
Abtion operates. Throughout your long career at Abtion, you have established
```
A first step can be to give more importance to the fact that they'd be owning the way Abtion operates.